### PR TITLE
add attach to pod reflection and switch to fork for vk library

### PR DESCRIPTION
**Pull request by aka-somix**
Co-authored-by: aka-somix <somix.land@null.net>
Co-authored-by: Pablo Borrelli <pablo.borrelli0@gmail.com>

# Description
Add support for Attach command to an offloaded pod. 
Also, virtual-kubelet library has been replaced with a local fork in liqotech org.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -45,19 +45,19 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.10.3
 	inet.af/netaddr v0.0.0-20220811202034-502d2d690317
-	k8s.io/api v0.25.5
-	k8s.io/apiextensions-apiserver v0.25.2
-	k8s.io/apimachinery v0.25.5
-	k8s.io/apiserver v0.25.5
+	k8s.io/api v0.26.0
```
[the remark this answers is not in the document]
Comment (ssoBAekiL) [reply go.mod:48]: We only needed to manually update sigs.k8s.io/controller-runtime from v013.1 to v0.14.1, all other updates are only the result of running 'go mod tidy' after the single manual update

**Review by aleoli — APPROVED** [review]

**Pull request merged**